lib/pharos/scripts/configure-kubelet-proxy.sh

```diff
@@ -37,8 +24,18 @@ spec:
   hostNetwork: true
 EOF
 
-systemctl daemon-reload
-systemctl restart kubelet
+if [ ! -e /etc/kubernetes/kubelet.conf ]; then
```

Comment:
Makes more sense to write this as it used to be: `if [ ! -e /usr/bin/kubelet ]`

Reply:
More sense = the block condition should be something that's satisfied by executing the block

Right now this is some side effect of the `JoinNode` phase, which happens after the `configure-kube.sh`... spooky action at a distance.